paddlehub/tokenizer/tokenizer.py

```diff
@@ -78,7 +78,11 @@ def get_vocab(self):
 
     def _convert_token_to_id(self, token):
         """ Converts a token (str) in an id using the vocab. """
-        return self.vocab.get(token, None)
+        v = self.vocab.get(token, None)
+        if v:
+            return v
+        else:
+            return 0
```

Comment:
To find OOV in vocabulary, it would be better to return None. The number 0 is also the index in  the vocabulary.

Reply:
But in tokenizer if it is none it will not get the id, so it may cause the term to be lost, is there a better way?

Reply:
1. We already do experiments and find out that drop the OOV term and replace it as [UNK] token make no difference on the model performance.
2. Replace OOV as vocab[0] doesn't make sense.
3. If replace OOV as [UNK] token, then some modules' vocabulary doesn't contain the  [UNK] token.

As refered, to find OOV in vocabulary, it would be better to return None.